app/models/shipit/stack.rb

```diff
@@ -96,18 +96,20 @@ def trigger_task(definition_id, user, env: nil)
         until_commit_id: commit.id,
         since_commit_id: commit.id,
         env: definition.filter_envs(env),
+        allow_concurrency: definition.allow_concurrency? || force,
```

Comment:
It doesn't really matter. Either the task already allow concurrency, and then `force` is pointless, or either it doesn't and then only `force` being `true` will change a thing.